x-pack/plugins/data_visualizer/public/application/data_drift/data_drift_app_state.tsx

```diff
@@ -77,6 +77,7 @@ export const DataDriftDetectionAppState: FC<DataDriftDetectionAppStateProps> = (
     unifiedSearch,
   } = getPluginsStart();
   const services = {
+    ...coreStart,
     data,
     maps,
```

Comment:
`coreStart` now has a `security` entry, so this is now a TS violation for conflicting key:

```ts
{
   security: something,
   ...coreStart,
}
```

I just reordered the spread.

Reply:
I think it's fine letting the owning team decide how they want to manage that. (tbh, the "let do a global service singleton with core service and all the plugin contracts we need and then use it everywhere" it a pattern I've been trying to discourage for years, so I'm not fixing tech debt I fought against)